src/test/java/com/google/cloud/datafusion/plugin/ariba/source/util/UtilTest.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright © 2021 Cask Data, Inc.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ * 
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.datafusion.plugin.ariba.source.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Junit test class for {@link Util} class methods
+ * 
+ * @author dhirajkumar
+ */
+public class UtilTest {
+
+  /**
+   * Test method for
+   * {@link com.google.cloud.datafusion.plugin.ariba.source.util.Util#isNotNullOrEmpty(CharSequence)}.
+   */
+  @Test
+  public void isNotNullOrEmptyString() {
```

Comment:
All test methods should start with `test`.
For eg. `testIsNotNullOrEmptyString()`

Reply:
Fixed